frontend/src/components/Entity.vue

```diff
@@ -91,12 +92,14 @@ export default {
         this.entity = null;
       }
     },
-    async onUpdate() {
-      await this.updateEntity();
+    async onUpdate(entity) {
+      if (entity) {
+        this.entity = entity;
+      }
```

Comment:
If I'm not missing something, If there is no entity emitted, the EntityForm was used for creation of a new entity and there is no need to update

Reply:
https://github.com/SUSE/aimaas/pull/201/files#diff-cdc34d04ddeed35251b6f776db2bd7a27384f0b8889803cda15f598a419bc2a4R267

If you want we can emit an update event if only in edit mode? I was kind of afraid doing this specifically, since I could break something which use it even in create mode

Reply:
There are also the deletion and restoration of deleted entities, which would provide a falsy `entity` argument. So, I would be inclined to follow your original approach :+1: